packages/nextclade-web/src/io/fetchSingleDatasetAuspice.ts

```diff
@@ -1,13 +1,24 @@
+import { isEmpty } from 'lodash'
 import { attrStrMaybe, AuspiceTree, Dataset } from 'src/types'
 import { removeTrailingSlash } from 'src/io/url'
-import { axiosFetch } from 'src/io/axiosFetch'
+import { axiosFetch, axiosFetchOrUndefined } from 'src/io/axiosFetch'
 
 export async function fetchSingleDatasetAuspice(datasetJsonUrl_: string) {
   const datasetJsonUrl = removeTrailingSlash(datasetJsonUrl_)
 
   const auspiceJson = await axiosFetch<AuspiceTree>(datasetJsonUrl, {
     headers: { Accept: 'application/json, text/plain, */*' },
   })
+
+  if (isEmpty(auspiceJson.root_sequence)) {
+    const sidecar = await axiosFetchOrUndefined<Record<string, string>>(datasetJsonUrl, {
+      headers: { Accept: 'application/vnd.nextstrain.dataset.root-sequence+json' },
```

Comment:
You might assert here that the response Content-Type is `application/vnd.nextstrain.dataset.root-sequence+json`, so that random JSON isn't shoved into the `root_sequence` field?

For example, consider what happens if the input URL is https://data.nextstrain.org/ncov_open_global_all-time.json...

![image](https://github.com/nextstrain/nextclade/assets/79913/4fce3a60-25d2-4333-8cdb-70d0c0eff62a)

Reply:
Alternatively, you could validate `sidecar` here against https://nextstrain.org/schemas/dataset/root-sequence.

Reply:
LGTM!

![image](https://github.com/nextstrain/nextclade/assets/79913/178cbf6f-dc40-48b0-b907-bce9cd0a4c66)